components/Main.vue

```diff
@@ -1,8 +1,274 @@
 <template>
-  <v-row>
-    <v-col class="text-center text-h3">
-      Let's go!
-      <v-icon color="success" icon="mdi-rocket-launch-outline" size="x-small" />
+  <v-row :class="[$vuetify.theme.dark ? 'currency-converter dark-theme' : 'currency-converter light-theme']">
+    <v-col cols="12" class="text-center pb-0">
+      <div class="d-flex">
+        <img src="../docs/logo-cd.png" alt="logo-accueil">
+        <h1>Convertisseur de devises</h1>
+      </div>
+      <div class="converter">
+        <v-text-field v-model="amount" label="Montant" type="number" class="amount" :rules="[v => v >= 0 ]" outlined />
+        <v-select v-model="fromCurrency" :items="currencyOptions" label="De" class="de" outlined />
+        <v-icon @click="swapCurrencies" class="icon-change" color="white">
+          mdi-swap-horizontal
+        </v-icon>
+        <v-select v-model="toCurrency" :items="currencyOptions" label="Vers" class="vers" outlined />
+      </div>
+      <div v-if="result" class="conversion-result mt-3">
+        <h4>{{ formattedResult }}</h4>
+        <p>{{ currentDate }}</p>
+      </div>
+    </v-col>
+    <v-col cols="12" class="switch-theme">
+      <v-switch v-model="$vuetify.theme.dark" label="Sombre/Clair" class="mt-4"></v-switch>
```

Comment:
Le tag peut s'auto-fermer.
```suggestion
      <v-switch v-model="$vuetify.theme.dark" label="Sombre/Clair" class="mt-4" />
```